### DISCUSSION TITLE
Meshnet support

### DISCUSSION THREAD

**Discussion by taimansulimani** [bubuntux/nordvpn#374] [Q&A]
I need a way to access my network remotely, I have no idea what to do to the compose file to make this happen. The new meshnet feature seems like it could help. Is meshnet support coming?

**Comment by renxida** [comment]
Hello! Same deal here. Any news?

**Comment by yeahmyusernameis32characterslong** [comment]
Meshnet support means you log in to NordVPN's website and create a manual connection (access tokens), but I am not sure what the format in the yaml script should be as NordVPN provides username and password tokens. The YAML is only expecting one entry for TOKEN=
